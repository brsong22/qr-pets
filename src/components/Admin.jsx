import React from 'react';
import AdminPetForm from './InputComponent/AdminPetForm';

export default () => (
  <div className="container">
    <h1 className="titleHeader">Adopt a pet today!</h1>
    <AdminPetForm />
  </div>
);
