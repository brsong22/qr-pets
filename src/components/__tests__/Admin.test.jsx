import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Admin from '../Admin';

describe('Admin', () => {
  it('renders as expected', () => {
    const wrapper = shallow(<Admin />);

    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
