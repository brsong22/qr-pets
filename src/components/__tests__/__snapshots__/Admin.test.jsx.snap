// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Admin renders as expected 1`] = `
<div
  className="container"
>
  <h1
    className="titleHeader"
  >
    Adopt a pet today!
  </h1>
  <AdminPetForm
    tags=""
  />
</div>
`;
